packages/eslint-plugin/src/rules/only-throw-error.ts

```diff
@@ -55,13 +65,14 @@ export default createRule<Options, MessageIds>({
   },
   defaultOptions: [
     {
+      allowThrowing: readonlynessOptionsDefaults.allow,
```

Comment:
[Refactor] The `readonlyness*` values are for readonly-related rules, not this one.

```suggestion
      allowThrowing: [],
```